backend/qemu.pm

```diff
@@ -186,8 +187,10 @@ sub open_file_and_send_fd_to_qemu {
     my ($self, $path, $fdname) = @_;
     my $rsp;
 
+    mkpath(dirname($path));
```

Comment:
As far as I know this won't throw an exception if it fails (in contrast to `File::Path::make_path` or `Mojo::File::make_path`). So either check the return value or use the one of the other functions.

Reply:
it's covered by "autodie"

Reply:
The fact that such an intrusive module exists seems insane but it is Perl after all so I'm not surprised.